shell_integration/nautilus/syncstate.py

```diff
@@ -86,9 +87,9 @@ def addListener(self, listener):
     def _connectToSocketServer(self):
         try:
             self._sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-            postfix = "/" + appname + "/socket"  # Should use os.path.join instead
-            sock_file = get_runtime_dir() + postfix
-            print ("Socket: " + sock_file + " <=> " + postfix)
+            postfix = os.path.join(appname, "socket")
```

Comment:
Does this work like you think?
It was `/appname/...` now it is probably `appname/...`: the leading `/` is not there?